cmd/debug.go

```diff
@@ -0,0 +1,17 @@
+package cmd
+
+import (
+	"github.com/qbarrand/debuggo/internal/debug"
+	"github.com/urfave/cli/v2"
+)
+
+var debugCommand = &cli.Command{
+	Name:   "debug",
+	Usage:  "Debug a program in a manner similar to gdb",
+	Action: db,
+}
+
+func db(c *cli.Context) error {
```

Comment:
Maybe we can find a better name for this function? How about `debug`?

Reply:
The module is named debug so it will clash. But I can maybe put dbg.